requirements/requirements-optionals.txt

```diff
@@ -2,3 +2,4 @@
 markdown==2.5.2
 django-guardian==1.3.2
 django-filter==0.10.0
```

Comment:
Okay. I'm wondering if we oughta catch that possibility in the encoder class. Eg if a tzinfo object is passed to the renderer, and `pytz` is not installed then raise an error? We wouldn't want `"<django.utils.timezone.LocalTimezone object at 0x105a5af90>" to be valid output from our API.